src/pkg/repository/repository.go

```diff
@@ -345,6 +266,68 @@ func (r repository) Counter() *count.Bus {
 	return r.counter
 }
 
+func (r *repository) addContextInfo(ctx context.Context) context.Context {
+	return clues.Add(
+		ctx,
+		"acct_provider", r.Account.Provider.String(),
+		"acct_id", clues.Hide(r.Account.ID()),
+		"storage_provider", r.Storage.Provider.String())
```

Comment:
clues will stringify values for you.  I assume this is from old code, so this is mostly informational for you.
```suggestion
		"acct_provider", r.Account.Provider,
		"acct_id", clues.Hide(r.Account.ID()),
		"storage_provider", r.Storage.Provider)
```